coalib/bearlib/aspects/Formatting.py

```diff
@@ -391,5 +392,11 @@ class docs:
         Choosing a preferred quotation and using it everywhere (if possible).
         """
     preferred_quotation = Taste[str](
-        'Represents the preferred quotation',
+        'Represents the preferred quotation marks.'
+        'It ensures that every string contains the selected style of quotes.',
         ('\'', '"'), default='\'')
+    force_preferred_quotation = Taste[bool](
```

Comment:
we've decided to remove this and go back to the fundamentals of literal strings and how languages differ.  So this should be removed for the moment, so that the other parts of this PR can get merged.

Reply:
I have removed `force_preferred_quotation`.

I have also updated the default value of `preferred_quotation` as we are doing it for string literals now and languages like CPP, C does not support `'` in strings. Moreover, that is what is default in the bear here at https://github.com/coala/coala-bears/blob/00c07fb1d4eeaaaa4cd76d6650b5ccfbb0f40be8/bears/general/QuotesBear.py#L66